dask/config.py

```diff
@@ -102,6 +102,25 @@ def merge(*dicts):
     return result
 
 
+def normalize_nested_keys(config):
+    """ Replaces underscores with hyphens for keys for a nested Mapping
+
+    Examples
+    --------
+    >>> a = {'x': 1, 'y_1': {'a_2': 2}}
+    >>> normalize_nested_keys(a)  # doctest: +SKIP
```

Comment:
Just curious, why is this doctest skipped?

Reply:
Good catch, thanks! I initially copied that example from another docstring in the `config.py` module and forgot to remove `# doctest: +SKIP`. Just pushed a commit with it removed.